ios/Classes/AblyFlutterPlugin.m

```diff
@@ -93,6 +93,64 @@ -(void)registerWithCompletionHandler:(FlutterResult)completionHandler;
     result(nil);
 };
 
+static FlutterHandler _attachRealtimeChannel = ^void(AblyFlutterPlugin *const plugin, FlutterMethodCall *const call, const FlutterResult result) {
```

Comment:
Rather confused as to why some local variables (pointers to Objective-C object instances) defined here are `*const` and others are not. Can you make them constant consistently unless there's a reason for the pointers to be mutable.

Reply:
@QuintinWillison here `result` is a function.

Reply:
That's irrelevant! What's to stop me writing code (somewhere) in scope of this static variable that does:

`_attachRealtimeChannel = NULL;`

❓ 

Also, it's a _block_, not a _function_. 😝

Reply:
Actually that's a new thing I've spotted (that `_attachRealtimeChannel` is mutable).

My comment was actually relating to the local variables defined within the block implementation.

Reply:
e.g. where you have:

```objc
ARTRealtime* realtimeWithHandle = [ably realtimeWithHandle: realtimeHandle];
```

You should make that pointer constant as it's never reassigned:

```objc
ARTRealtime *const realtimeWithHandle = [ably realtimeWithHandle: realtimeHandle];
```

Reply:
And for this particular line:

```suggestion
static const FlutterHandler _attachRealtimeChannel = ^void(AblyFlutterPlugin *const plugin, FlutterMethodCall *const call, const FlutterResult result) {
```

Reply:
Cool, thanks for the details.

Addressed all `const`'s here 72c71c87f575cf162fa7e229d52fced612cb2099